src/cli/utils/utils.toit

```diff
@@ -108,9 +108,9 @@ download-url url/string --out-path/string --ui/Ui -> none:
   ui.info "Downloading $url."
 
   network := net.open
+  certificate-roots.install-all-trusted-roots
```

Comment:
I kinda want to this a bit earlier. That's typically how we deal with installed certificates.

Reply:
done.